app/views/logix/contribute.html.erb

```diff
@@ -15,6 +15,10 @@
                         Contact us @
                         <a href="mailto:support@circuitverse.org" target="_top">support@circuitverse.org</a>
                       </p>
+                      <p>
+                        Join our chat room @
+                        <a href="https://gitter.im/CircuitVerse/community" target="_top">Gitter Channel</a>
```

Comment:
Can we open the chat room in a new tab using `_blank`. Thanks

Reply:
Yes sir will do so
Thanks for pointing out the issue number will change it too :+1:

Reply:
Hi, no sir please. Haven't received my knighthood (yet).

Reply:
@armudgal Ok :smile:
I have also included the chat link in the about page.